src/input/plugins/CurlInputPlugin.cxx

```diff
@@ -431,9 +434,11 @@ CurlInputStream::InitEasy()
 		request->SetOption(CURLOPT_PROXYUSERPWD,
 				   StringFormat<1024>("%s:%s", proxy_user,
 						      proxy_password).c_str());
+        if (cacert != nullptr)
+            request->SetOption(CURLOPT_CAINFO, cacert);
 
-	request->SetVerifyPeer(verify_peer);
-	request->SetVerifyHost(verify_host);
+	request->SetOption(CURLOPT_SSL_VERIFYPEER, verify_peer ? 1L : 0L);
+	request->SetOption(CURLOPT_SSL_VERIFYHOST, verify_host ? 2L : 0L);
```

Comment:
Why this change?

Reply:
Sorry, started with 0.22.4, swithed to main, so there is this change in the last commit